src/components/GameInput.js

```diff
@@ -18,30 +14,32 @@ const StyledInputButton = styled.button`
   fill: ${props => props.theme.brightFont};
   height: 80px;
   width: 140px;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
 `;
 
-function GameButton({ onClick, direction }) {
-  if (direction === "right") {
+function GameButton({ handleTap, handleRelease, direction }) {
+  const { height, width } = useWindowDimensions();
+
+  console.log(width > 420);
+  if (width > 420) {
     return (
-      <StyledInputButton onClick={onClick}>
-        <Pointer white />
+      <StyledInputButton onMouseDown={handleTap} onMouseUp={handleRelease}>
```

Comment:
Just curious: The "onMouseDown" also applies to mobile versions when there is no actual cursor?

Reply:
Nope, for that case i use the onTouchStart and onTouchEnd events. Was that change not included here? :(